automatic/sql-server-data-tools/sql-server-data-tools.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>sql-server-data-tools</id>
```

Comment:
This is not SQL Server Data Tools (which was the name of the whole installer) but only the extension for integration services projects. I would call it `ssis-vs2019`.